src/pfe/portal/modules/User.js

```diff
@@ -866,17 +863,14 @@ module.exports = class User {
   /**
    * Function to remove the docker registries from PFE
    */
-  async removeRegistrySecret(url) {
+  async removeRegistrySecret(address) {
     const registrySecretList = [];
 
     try {
       // Handle dockerhub specifically on local since docker.io does not work
-      if (url === "docker.io" && !global.codewind.RUNNING_IN_K8S) {
-        // eslint-disable-next-line no-param-reassign
-        url = "https://index.docker.io/v1/";
-      } else if (!url.startsWith("https://") && !url.startsWith("http://")) {
+      if (address === "docker.io" && !global.codewind.RUNNING_IN_K8S) {
         // eslint-disable-next-line no-param-reassign
-        url = "https://" + url;
+        address = "https://index.docker.io/v1/";
```

Comment:
Should this be ended with / ? I think in one of the places we are doing address + '/' + namespace.

Reply:
yes we're doing that when we read in address `imagePushRegistry = workspaceSettingsInfo.registryAddress.replace(/\/\s*$/, "") + "/" + workspaceSettingsInfo.registryNamespace.trim();`